nsiqcppstyle_exe.py

```diff
@@ -78,91 +77,138 @@ def Usage():
                 to create one (e.g., in a read-only file system)
   --var=key: value,key: value
```

Comment:
This help form is misleading because the space between "key:" and the value is mandatory.  Under Windows (and possibly Linux) not quoting the pair will lead to parsing errors.  I recommend changing the line to the following:

--var="key: value,key: value"

or something to that effect.  See my comment below regarding --var.

Reply:
In my experience, under both Windows and Linux, I need to single/double quote the --var string.  You didn't relate to my comment.  Is this proper procedure?

Reply:
Please check the code again